website/cue/reference/components/sources/base/fluent.cue

```diff
@@ -46,6 +46,15 @@ base: components: sources: fluent: configuration: {
 			type: uint: unit: "seconds"
 		}
 	}
+	permit_origin: {
+		description: """
+			List of allowed origin IP networks
+
+			By default, all origins are allowed
```

Comment:
```suggestion
			List of allowed origin IP networks.

			By default, all origins are allowed.
```

Reply:
Thanks. I have added a period in the shared docs now.